src/main/webapp/components/ControlWrapper.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import ControlGroup from '@splunk/react-ui/ControlGroup';
 import styled from 'styled-components';
 
+import MarkDownMessage from './MarkDownMessage';
```

Comment:
Rename this component from "MarkDownMessage" to "MarkdownMessage" for consistency.